WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -125,37 +137,9 @@ private extension OrderTableViewCell {
 
 private extension OrderTableViewCell {
     func configureBackground() {
-        backgroundColor = .listForeground(modal: false)
-        let backgroundView: UIView = {
-            let view = UIView()
-            view.backgroundColor = .listBackground
-            let separatorHeight: CGFloat = 1
-
-            let topSeparatorView = UIView()
-            topSeparatorView.backgroundColor = .border
-            topSeparatorView.translatesAutoresizingMaskIntoConstraints = false
-            view.addSubview(topSeparatorView)
-            NSLayoutConstraint.activate([
-                view.topAnchor.constraint(equalTo: topSeparatorView.topAnchor),
-                view.leadingAnchor.constraint(equalTo: topSeparatorView.leadingAnchor),
-                view.trailingAnchor.constraint(equalTo: topSeparatorView.trailingAnchor),
-                topSeparatorView.heightAnchor.constraint(equalToConstant: separatorHeight)
-            ])
-
-            let bottomSeparatorView = UIView()
-            bottomSeparatorView.backgroundColor = .border
-            bottomSeparatorView.translatesAutoresizingMaskIntoConstraints = false
-            view.addSubview(bottomSeparatorView)
-            NSLayoutConstraint.activate([
-                view.bottomAnchor.constraint(equalTo: bottomSeparatorView.bottomAnchor, constant: separatorHeight/2),
-                view.leadingAnchor.constraint(equalTo: bottomSeparatorView.leadingAnchor),
-                view.trailingAnchor.constraint(equalTo: bottomSeparatorView.trailingAnchor),
-                bottomSeparatorView.heightAnchor.constraint(equalToConstant: separatorHeight/2)
-            ])
-            view.backgroundColor = .wooCommercePurple(.shade0)
-            return view
-        }()
-        selectedBackgroundView = backgroundView
+        var backgroundConfiguration = UIBackgroundConfiguration.listPlainCell()
+        backgroundConfiguration.backgroundColor = .listBackground
+        self.backgroundConfiguration = backgroundConfiguration
```

Comment:
From what I can tell, I think we can delete this method... do you think it's needed for anything?

Reply:
We still use it for the initial cell configuration, otherwise the configuration background for the `.listBackground` colour is not applied, that said this is not really perceptible (if at all), since the only difference is the usage of `.white` vs `.gray(.shade0)` in light mode.

I plan to look into the APIs you shared in the other comment ([contentConfiguration](https://developer.apple.com/documentation/uikit/uitableviewcell/3601057-contentconfiguration#discussion)) and see if we can simplify further the cells that we'll be touching with the project 👍